XenOvfTransport/Export.cs

```diff
@@ -354,6 +355,37 @@ private EnvelopeType _export(Session xenSession, string targetPath, string ovfna
                         OVF.AddOtherSystemSettingData(ovfEnv, vsId, "vgpu", sb.ToString(), OVF.GetContentMessage("OTHER_SYSTEM_SETTING_DESCRIPTION_4"));
                     }
                 }
+
+                var allProxies = PVS_proxy.get_all_records(xenSession).Select(kvp => kvp.Value).ToList();

```

Comment:
Why do you use a xapi call (get_all_records) for PVS proxies, but we use Resolve for the VIF and VM? Can't we also use ResolveAll for proxies?